src/MOI_wrapper.jl

```diff
@@ -871,7 +871,7 @@ function _fill_quad_Q(model, dQv, dQi, dQj, index_map)
     for ((vi1, vi2), val) in dict_dQ
         i = index_map[vi1].value
         j = index_map[vi2].value
-        for (vi, val) in dict
+        for (vi, val) in dict_dQ
```

Comment:
ok this is weird that this wasn't caught by tests before

Reply:
@joaquimg should there be a test where `dict_dQ` gets used?

Reply:
yep. i guess with more examples common, such bugs (if any) will surface